tce/opset/base/base.cc

```diff
@@ -2334,12 +2334,15 @@ END_DEFINE_STATE
 OPERATION_WITH_STATE(HWLOOP, HLOOP_SETUP)
 TRIGGER
     // sanity check
-    if (STATE.hwloopEnabled) {
-        RUNTIME_ERROR("Already hwloop is in progress");
```

Comment:
-> Hardware loop execution already in progress.

Reply:
Done